images/macos/provision/core/postgresql.sh

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash -e -o pipefail
 source ~/utils/utils.sh
 
-#Install latest version of postgresql
-brew_smart_install "postgres"
+# Fetch postgresql version to install from the toolset
+toolsetVersion=$(get_toolset_value '.postgresql.version')
 
-#Service postgresql should be started before use.
+# Search for the latest version of the postgresql corresponding to version from the toolset
+versionToInstall=$(brew search /postgresql@$toolsetVersion/ | awk -F' ' '{print $1}' | tail -1)
```

Comment:
Why can't we use just `brew_smart_install postgresql@$versionToInstall`?

Reply:
Indeed. I think it's just a precaution. But if you speak so too, we can omit it safely.

Reply:
Updated